server/libs/database/src/entities/asset.entity.ts

```diff
@@ -68,6 +69,10 @@ export class AssetEntity {
   @ManyToMany(() => TagEntity, (tag) => tag.assets, { cascade: true })
   @JoinTable({ name: 'tag_asset' })
   tags!: TagEntity[];
+
+  @ManyToMany(() => SharedLinkEntity, (link) => link.assets, { cascade: true })
+  @JoinTable({ name: 'shared_link__asset' })
```

Comment:
Is the double underscore on purpose or is it a typo? If it is on purpose, why? It reminds me of some weird css class naming convention :laughing:

Reply:
It is on purpose, thinking shared_link table with asset table so if single underscore it wouldn't have that distinction

Reply:
I'm not sure I follow. We have other tables with relationships like this and they all use single understand don't they?